Suggest/TagSuggest.ts

```diff
@@ -0,0 +1,21 @@
+import {TFolder} from "obsidian";
+import {TextInputSuggest} from "./suggest";
+
+export class TagSuggest extends TextInputSuggest<string> {
+	getSuggestions(inputStr: string): string[] {
+		// TS ignore is required here because getTags does not exist in the declarations but is a valid function.
+		// Source: https://forum.obsidian.md/t/efficiently-get-all-tags-through-the-api/38400/3
+		// @ts-ignore
+		return Object.keys(app.metadataCache.getTags()).filter((tag: string) => tag.toLowerCase().includes(inputStr.toLowerCase()));
```

Comment:
Can we remove the `#` from the listed tags and the search altogether? i.e. instead of listing "#tag" and "#mytag", the user would see "tag" and "mytag", then search like "mytag".